app/assets/javascripts/cardboard/new_tab_links.js

```diff
@@ -0,0 +1,10 @@
+$(document).on("page:update", function() {
+  $("a").click(function() {
+    var link_host = this.href.split('/')[3];
+    var document_host = document.location.href.split('/')[3];
```

Comment:
`.host` gets the main website host, which is slightly different than what we're looking it. I think an example illustrates it best.

Let's take `www.thesite.com/cardboard/offerings` as our document location. A link on the page directs us to `www.thesite.com/offerings/2`. Our criterion states we want this link to open in a new tab, since we're leaving Cardboard.

However, if we use `host`, both our values will be `thesite.com`, and the "open this in a new tab" part will not trigger. We need to compare the part of the string after the `.com/`, and check that those are the same. That's how CB is set up. Unfortunately, I think this is the way to do it -- but I wouldn't be surprised if JS had some way to grab that information. I don't know what it'd be called, though.